samples/client/petstore/java/jersey3/docs/AllOfRefToDouble.md

```diff
@@ -0,0 +1,14 @@
+
+
+# AllOfRefToDouble
+
+Object with allOf ref to double
+
+## Properties
+
+| Name | Type | Description | Notes |
+|------------ | ------------- | ------------- | -------------|
+|**height** | **Double** | Height as double |  [optional] |
+
+
+
```

Comment:
there shouldn't be changes to the java samples.

i'll clean that up in another pr